common/menu.c

```diff
@@ -666,6 +723,18 @@ void menuLoop(void) {
         }
 
         drawButtons(menu, false, &menupath_x_endpos);
+
+        if(active_entry != NULL && active_entry->type != ENTRY_TYPE_THEME) {
```

Comment:
Space after if. In addition, `!= NULL` is redundant (non-NULL is already true), you can leave it out.

Reply:
Probably should check for `type == ENTRY_TYPE_FILE` (likewise elsewhere), assuming supporting directories isn't intended.

Reply:
Supporting directories was intended actually - personally I occasionally press B by accident and end up in the SD root, having the "switch" directory starred at the beginning of the list is handy.

Reply:
ok

Reply:
Space added, != NULL removed. Let me know if you'd rather I remove support for directories.